### ISSUE TITLE
Gadfly#master fails

### ISSUE THREAD

**Issue by montyvesselinov** [GiovineItalia/Gadfly.jl#1205]
```
               _
   _       _ _(_)_     |  A fresh approach to technical computing
  (_)     | (_) (_)    |  Documentation: https://docs.julialang.org
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 0.7.0 (2018-08-08 06:46 UTC)
 _/ |\__'_|_|_|\__'_|  |  Official http://julialang.org/ release
|__/                   |  x86_64-apple-darwin14.5.0

(v0.7) pkg> test Gadfly
   Testing Gadfly
 Resolving package versions...
    Status `/var/folders/3z/rj6y6gfx4nd3h867q_mvjnw0000b9r/T/tmp77Dmoh/Manifest.toml`
  [621f4979] AbstractFFTs v0.3.2
  [7d9fca2a] Arpack v0.2.3
  [13072b0f] AxisAlgorithms v0.3.0
  [9e28174c] BinDeps v0.8.10
  [b99e7846] BinaryProvider v0.5.1
  [e1450e63] BufferedStreams v1.0.0
  [336ed68f] CSV v0.4.1
  [159f3aea] Cairo v0.5.6
  [49dc2e85] Calculus v0.4.1
  [324d7699] CategoricalArrays v0.4.0
  [944b1d66] CodecZlib v0.5.0
  [3da002f7] ColorTypes v0.7.5
  [5ae59095] Colors v0.9.4
  [bbf7d656] CommonSubexpressions v0.2.0
  [34da2185] Compat v1.2.0
  [a81c6b42] Compose v0.7.0
  [8f4d0f93] Conda v1.0.2
  [d38c429a] Contour v0.5.1
  [7ad07ef1] CoupledFields v0.1.0
  [a93c6f00] DataFrames v0.14.1
  [9a8bc11e] DataStreams v0.4.1
  [864edb3b] DataStructures v0.14.0
  [01453d9d] DiffEqDiffTools v0.6.0
  [163ba53b] DiffResults v0.0.3
  [b552c78f] DiffRules v0.0.7
  [b4f34e82] Distances v0.7.3
  [31c24e10] Distributions v0.16.4
  [ffbed154] DocStringExtensions v0.5.0
  [8f5d6c58] EzXML v0.9.0
  [7a1cc6ca] FFTW v0.2.4
  [5789e2e9] FileIO v1.0.2
  [53c48c17] FixedPointNumbers v0.5.3
  [f6369f11] ForwardDiff v0.9.0
  [c91e804a] Gadfly v0.8.0+ #master (https://github.com/GiovineItalia/Gadfly.jl.git)
  [a2bd30eb] Graphics v0.4.0
  [0862f596] HTTPClient v0.2.1
  [a1b4810d] Hexagons v0.2.0
  [d9be37ee] Homebrew v0.7.0
  [9b13fd28] IndirectArrays v0.5.0
  [a98d9a8b] Interpolations v0.10.3
  [c8e1da08] IterTools v1.1.0
  [82899510] IteratorInterfaceExtensions v0.1.1
  [682c06a0] JSON v0.19.0
  [e5e0dc1b] Juno v0.5.3
  [5ab0869b] KernelDensity v0.5.0
  [b27032c2] LibCURL v0.4.1
  [522f3ed2] LibExpat v0.5.0
  [2ec943e9] Libz v1.0.0
  [d3d80556] LineSearches v7.0.0
  [4345ca2d] Loess v0.5.0
  [1914dd2f] MacroTools v0.4.4
  [442fdcdd] Measures v0.3.0
  [e89f7d12] Media v0.4.1
  [e1d29d7a] Missings v0.3.1
  [78c3b35d] Mocking v0.5.7
  [d41bc354] NLSolversBase v7.1.0
  [77ba4419] NaNMath v0.3.2
  [4d1e1d77] Nullables v0.0.8
  [6fe1bfb0] OffsetArrays v0.8.1
  [429524aa] Optim v0.16.0
  [bac558e1] OrderedCollections v1.0.2
  [90014a1f] PDMats v0.9.5
  [d96e819e] Parameters v0.10.1
  [69de0a69] Parsers v0.2.7
  [85a6dd25] PositiveFactorizations v0.2.1
  [1fd47b50] QuadGK v2.0.2
  [df47a6cb] RData v0.5.0
  [ce6b1742] RDatasets v0.5.0
  [c84ed2f1] Ratios v0.3.0
  [189a3867] Reexport v0.2.0
  [ae029012] Requires v0.5.2
  [79098fc4] Rmath v0.5.0
  [992d4aef] Showoff v0.2.1
  [a2af1166] SortingAlgorithms v0.3.1
  [276daf66] SpecialFunctions v0.7.1
  [90137ffa] StaticArrays v0.8.3
  [2913bbd2] StatsBase v0.25.0
  [4c63d2b9] StatsFuns v0.7.0
  [3783bdb8] TableTraits v0.3.1
  [bd369af6] Tables v0.1.9
  [f269a46b] TimeZones v0.8.0
  [3bb67fe8] TranscodingStreams v0.8.1
  [30578b45] URIParser v0.4.0
  [81def892] VersionParsing v1.1.3
  [ea10d353] WeakRefStrings v0.5.3
  [c17dfb99] WinRPM v0.4.2
  [efce3f68] WoodburyMatrices v0.4.1
  [2a0f44e3] Base64  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Base64`]
  [ade2ca70] Dates  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Dates`]
  [8bb1440f] DelimitedFiles  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/DelimitedFiles`]
  [8ba89e20] Distributed  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Distributed`]
  [9fa8497b] Future  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Future`]
  [b77e0a4c] InteractiveUtils  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/InteractiveUtils`]
  [76f85450] LibGit2  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/LibGit2`]
  [8f399da3] Libdl  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Libdl`]
  [37e2e46d] LinearAlgebra  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/LinearAlgebra`]
  [56ddb016] Logging  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Logging`]
  [d6f4376e] Markdown  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Markdown`]
  [a63ad114] Mmap  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Mmap`]
  [44cfe95a] Pkg  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Pkg`]
  [de0858da] Printf  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Printf`]
  [9abbd945] Profile  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Profile`]
  [3fa0cd96] REPL  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/REPL`]
  [9a3f8284] Random  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Random`]
  [ea8e919c] SHA  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/SHA`]
  [9e88b42a] Serialization  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Serialization`]
  [1a1011a3] SharedArrays  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/SharedArrays`]
  [6462fe0b] Sockets  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Sockets`]
  [2f01184e] SparseArrays  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/SparseArrays`]
  [10745b16] Statistics  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Statistics`]
  [4607b0f0] SuiteSparse  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/SuiteSparse`]
  [8dfed614] Test  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Test`]
  [cf7118a7] UUIDs  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/UUIDs`]
  [4ec0a83e] Unicode  [`/Applications/Julia-0.7.app/Contents/Resources/julia/bin/../share/julia/stdlib/v0.7/Unicode`]
[ Info: Loading Cairo backend into Compose.jl
ERROR: LoadError: GitError(Code:ENOTFOUND, Class:Repository, could not find repository from '/Users/monty/.julia/packages/Gadfly/TkK62')
Stacktrace:
 [1] GitRepo(::String) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v0.7/LibGit2/src/error.jl:101
 [2] top-level scope at none:0
 [3] include at ./boot.jl:317 [inlined]
 [4] include_relative(::Module, ::String) at ./loading.jl:1038
 [5] include(::Module, ::String) at ./sysimg.jl:29
 [6] include(::String) at ./client.jl:398
 [7] top-level scope at none:0
in expression starting at /Users/monty/.julia/packages/Gadfly/TkK62/test/runtests.jl:10
ERROR: Package Gadfly errored during testing
```

**Comment by tlnagy** [comment]
That's a really weird error that looks like the Gadfly compiled code is missing. Did you try running `] build Gadfly` and try again?

**Comment by tlnagy** [comment]
Should be fixed by #1213. Please update your Gadfly master and open a new issue if this isn't fixed. Thanks.
